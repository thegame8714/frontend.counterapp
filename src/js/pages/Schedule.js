import React, { Component } from "react";
import PageHeading from '../components/PageHeading';

class Schedule extends Component {
	render() {
		return (
			<div className='content'>
				<PageHeading value="Schedule page" />
			</div>
		);
	}
}

export default Schedule;
