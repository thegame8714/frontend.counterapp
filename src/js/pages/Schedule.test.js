import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Schedule from './Schedule';

describe('Schedule Page', () => {
  it('should render page tile as H1 with text', () => {
    const route = { pageTitle: "Test Schedule page title" }
    const wrapper = mount(<Schedule route={route} />);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});