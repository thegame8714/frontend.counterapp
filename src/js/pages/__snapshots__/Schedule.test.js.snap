exports[`Schedule Page should render page tile as H1 with text 1`] = `
<Schedule
  route={
    Object {
      "pageTitle": "Test Schedule page title",
    }
  }>
  <div
    className="content">
    <PageHeading
      value="Schedule page">
      <h1>
        Schedule page
      </h1>
    </PageHeading>
  </div>
</Schedule>
`;
